lib/src/base-styles/_nx-globals.scss

```diff
@@ -31,9 +30,8 @@ $-nx-scrollable-default-height: 400px;
   }
 
   &:focus {
-    border-color: var(--nx-color-interactive-border-focus);
     border-radius: var(--nx-border-radius);
-    outline: 0;
+    outline: 1px solid var(--nx-color-interactive-border-focus);
```

Comment:
I think `outline-offset` is needed to get the outline to the desired space from the text.

Reply:
> > Ah, I forgot to post the screenshots. 4px caused it to run over the adjacent text. 2px looked correct.
> 
> Are there adjustments that need to me made to the mockups to match that then?

I measured screenshots and it's `16px` in the RSC and `15px` in Sketch. Measuring from the underline to the edge of the icon. So near enough.